AdyenDropIn/Utilities/ComponentManager+PaymentComponentBuilder.swift

```diff
@@ -419,7 +433,7 @@ extension ComponentManager: PaymentComponentBuilder {
         )
     }
 
-    private func createBoletoComponent(_ paymentMethod: BoletoPaymentMethod) -> BoletoComponent {
+    func createBoletoComponent(_ paymentMethod: BoletoPaymentMethod) -> BoletoComponent {
```

Comment:
We have a linter rule to prevent us from NOT having a scope defined, right?

Reply:
Maybe we can turn swiftlint warnings into errors so every time one doesn't need to even focus on this during code review.